.eslintignore

```diff
@@ -8,3 +8,14 @@ packages/neutrino/test/fixtures/test-module/errorMiddleware.js
 # RC files. We then negate this ignore so they are still linted, but no longer
 # used as configuration by our development lint command.
 !packages/create-project/bin/commands/init/templates/*/.eslintrc.js
+
+# Ignore directories that may potentially have JS we don't need linted
+coverage
+docs
+build
+
+# Lint dotfiles
```

Comment:
Should we use the wildcard form here, to save us forgetting to add new files? eg:
`!.*.js`